### DISCUSSION TITLE
Práctica pastelería HTML y CSS

### DISCUSSION THREAD

**Discussion by OscarPedroza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#290] [Show and tell]
Buenas noches compañeros, alcance a terminar la práctica de la pastelería y me gustaría compartir la página, la realice con la ayuda de flexbox.

Este es el link: [https://oscarpedroza.github.io/practica-html-frontend-mission.github.io/pasteleria-clientes/](https://oscarpedroza.github.io/practica-html-frontend-mission.github.io/pasteleria-clientes/)

**Comment by hackit92** [comment]
Está excelente tu sitio compañero! Me parece muy completo tu despliegue, el único detalle que no encontré fue el dashboard para el pastelero en donde se ve el inventario, felicidades por tan buena práctica!

**Reply by OscarPedroza** [reply]
Muchas gracias compañero, tienes razón, separe la parte administrativa para tener un poco más separado el cliente y la información del pastelero,  Este es el link:[https://oscarpedroza.github.io/practica-html-frontend-mission.github.io/pasteleria-administrador/index.html]( https://oscarpedroza.github.io/practica-html-frontend-mission.github.io/pasteleria-administrador/index.html)

**Comment by JuanSalvador73** [comment]
Excelente trabajo. :D

**Reply by OscarPedroza** [reply]
Muchas gracias compañero!

**Comment by lunasaurio** [comment]
aprovecho este espacio para compartirles el mio, agradezco sus comentarios:
https://lunasaurio.github.io/
